content/gotraining.slide

```diff
@@ -713,11 +748,107 @@ Use `gofmt` or `goimport` as part of your editor/IDE setup
 [[https://godoc.org/gopkg.in/square/go-jose.v2][go-jose]] - JWT utils
  ...
 
-* Project work
+
+* A RESTful API starter (1/3)
+
+[[https://goplay.space/#YJqaOpvxiKT][Full basic sample]]
+[[https://goplay.space/#aXy28QMEiNK][Full sample]] with error handling
```

Comment:
not sure how i missed this the first time, but these don't run in the playground because it doesn't support third-party imports (chi). Perhaps a note alongside the links to let them know not to expect this to work, and to build the sample locally themselves.